app/views/admin/creator_settings/_form.html.erb

```diff
@@ -8,20 +8,24 @@
 </div>
 
 <div class="crayons-field mt-6 align-left">
-  <%= label_tag :logo_svg, class: "crayons-field__label" do %>
+  <div class="crayons-field__label relative">
     Logo
     <span class="crayons-field__required crayons-tooltip" data-tooltip="This will set the logo for your Forem" aria-describedby="logo-subtitle"></span>
```

Comment:
For this to work, it would need to be changed to 
```suggestion
    <span class="crayons-field__required crayons-hover-tooltip" data-tooltip="This will set the logo for your Forem"></span>
```

and I think the `aria-describedby` wouldn't help in this situation, as a hover only tooltip isn't accessible to keyboard navigation (TBH I wonder if we need it all - it feels to me like all the necessary info is already presented to the user)

Reply:
I'm going to go ahead and remove it.